endpoint_storage/src/claims.rs

```diff
@@ -0,0 +1,54 @@
+use serde::{Deserialize, Serialize};
+use std::fmt::Display;
+use utils::id::{TenantId, TimelineId};
+
+pub type EndpointId = String; // If needed, reuse small string from proxy/src/types.rc
```

Comment:
If you want to use this alias, please make a PR to add it to `utils::id` like `TenantId` and `TimelineId`. I like the suggestion, but this is not the place to make such an alias.

Reply:
Since this PR would be a one-liner, I believe it's safe to stack to this PR and reuse the alias from utils:: here